platforms/aws/giantnetes/main.tf

```diff
@@ -119,15 +119,35 @@ locals {
   }
 }
 
-# Generate ignition config.
-data "gotemplate" "bastion" {
-  template = "${path.module}/../../../templates/bastion.yaml.tmpl"
-  data     = "${jsonencode(local.ignition_data)}"
+# Render files before referencing them in main state
+# It should be applied before main apply
+# terraform apply -target="module.ignition" ./ 
```

Comment:
this will be added into bootstrap.sh in installations repository. I didn't find any other way how to overcome this.